phpBB/styles/prosilver/template/ucp_profile_signature.html

```diff
@@ -24,6 +24,7 @@ <h3>{L_SIGNATURE_PREVIEW}</h3>
 	<!-- INCLUDE posting_editor.html -->
 	<h3>{L_OPTIONS}</h3>
 	<fieldset>
+		<!-- EVENT ucp_profile_signature_posting_editor_options_prepend -->
```

Comment:
Twig syntax is preferred also for template events now.

Reply:
@3D-I Is there any place where I can read that?

I used old template to avoid mixing old template syntax and Twig.

Reply:
https://github.com/phpbb/phpbb/pull/5386

https://github.com/phpbb/phpbb/pull/5484#issuecomment-450023609